### DISCUSSION TITLE
Remix config watchPaths callback

### DISCUSSION THREAD

**Discussion by nadiTime** [remix-run/remix#3979] [Proposals]
I am using watchPaths to monitor changes in a component library I am building,
The problem I see is that whenever the library build and remix reloads I get the warning:
`Warning: Text content did not match. Server: "XXXXX" Client: "YYYYYY"`

How can I get rid of this? not that critical but it is just annoying.
Can we add an option to pass a callback to watchPaths?

**Comment by kiliman** [comment]
The "Text content did not match" warning, I'm not sure is coming from using `watchPaths`. Typically hydration warning are due to some external extension injecting markup prior to hydration.

Try running your site in *incognito* to see if one of your browser extensions are causing an issue.

**Reply by nadiTime** [reply]
Seems like the error still shows in incognito.
Any suggestions?
I did had the error `Warning: Did not expect server HTML to contain a <div> in <body>` which doesn't show in incognito...
